### DISCUSSION TITLE
Vignette color

### DISCUSSION THREAD

**Discussion by forerunrun** [pmndrs/postprocessing#531] [Ideas]
Hi I'm wondering if there were already limitations discovered on adding a colorIn uniform to the Vignette effect? It could be great to be able to change the color of the vignette from black to say white for neomorphism styles... As well as other colors for different use cases such as "bleeding" or "damage" like in retro games such as doom etc...

**Comment by vanruesc** [comment]
Hi,

white Vignette can be achieved by changing the `VignetteEffect.technique` to `VignetteTechnique.ESKIL` and adjusting the parameters. Adding a color option to `VignetteEffect` seems fine to me, but I think it would be better to create new effects for the other use cases you mentioned.

**Comment by vanruesc** [comment]
It's been a while since this was asked, but the `VignetteEffect` in the current v7 beta now has a color setting: https://github.com/pmndrs/postprocessing/blob/ed2dd70c46c1c3cd7f66f5877bfb2ad8166fa10c/src/effects/VignetteEffect.ts#L45

To get the colored Vignette to render as expected, it should be applied after a `ToneMappingEffect`.
